deployment/helm/node-feature-discovery/values.yaml

```
@@ -10,7 +10,8 @@ nameOverride: ""
 fullnameOverride: ""
 namespaceOverride: ""
 
-enableNodeFeatureApi: true
+featureGate:
```

Oh, one more suggestion:

```suggestion
featureGates:
```